EarlGrey/Exception/GREYDefaultFailureHandler.m

```diff
@@ -52,11 +52,12 @@ - (void)handleException:(GREYFrameworkException *)exception details:(NSString *)
   }
 
   NSMutableString *log = [[NSMutableString alloc] init];
-  // Start on fresh new line.
-  [log appendString:@"\n"];
```

Comment:
No. Log output which was put into a format string in XCTestCase+GREYAdditions is now in this file.

A new line was used here before because it put a blank line between different strings being put in a format string in XCTestCase+GREYAdditions, but not in front of the first string.

The first string originally in XCTestCase+GREYAdditions never had a blank line in front of it. It must not have one, because the first line is shown as explanation of error by Xcode, and if it only has a new line the message will be blank. (Example of this message: http://iosunittesting.com/wp-content/uploads/TestAssistantExample.png)